handlers/middlewares.go

```diff
@@ -101,12 +101,13 @@ func (h *Handler) SessionInjectorMiddleware(next func(http.ResponseWriter, *http
 			logrus.Warn("unable to read session from the session persister, starting with a new one")
 		}
 
-		if prefObj == nil {
-			prefObj = &models.Preference{
-				AnonymousUsageStats:  true,
-				AnonymousPerfResults: true,
-			}
-		}
+		// if prefObj == nil {
```

Comment:
Why?

Reply:
The method I have used in place of this is handling this case.

Reply:
No actually, this piece of code( including the below handler ) is already invoked in the session-sync handler and because of this, the config is detected now. How will adding it here make any difference when it is already being invoked once?

Reply:
Can we remove the comments then?

Reply:
Okay